.bazelrc

```diff
@@ -262,7 +262,7 @@ build:remote --strategy=Genrule=remote,sandboxed,local
 build:remote --remote_timeout=7200
 build:remote --google_default_credentials=true
 build:remote --remote_download_toplevel
-build:remote --nobuild_runfile_links
+# build:remote --nobuild_runfile_links
```

Comment:
unintentional check in?

Reply:
undid this just now, sorry for the confusion!  i was just trying out undoing a phlax change that was causing CI to fail.  phlax is going to send in a fix for it tomorrow.  thanks for reviewing!